### ISSUE TITLE
Dependency on futures package breaks Python 3

### ISSUE THREAD

**Issue by ianhelle** [VirusTotal/vt-graph-api#35]
In setup.py the install_requires includes futures.
This is a Python 2 backport of Python 3 functionality, which is all in the std library.
Doing anything with this package (including trying to build a wheel/egg for it) causes a failure in Python 3

```
Collecting futures
  Using cached futures-3.0.5.tar.gz (25 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [25 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 14, in <module>
        File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/setuptools/__init__.py", line 18, in <module>
          from setuptools.dist import Distribution
        File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/setuptools/dist.py", line 31, in <module>
          from setuptools.extern.more_itertools import unique_everseen
        File "<frozen importlib._bootstrap>", line 991, in _find_and_load
        File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
        File "<frozen importlib._bootstrap>", line 657, in _load_unlocked
        File "<frozen importlib._bootstrap>", line 556, in module_from_spec
        File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/setuptools/extern/__init__.py", line 52, in create_module
          return self.load_module(spec.name)
        File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/setuptools/extern/__init__.py", line 37, in load_module
          __import__(extant)
        File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/setuptools/_vendor/more_itertools/__init__.py", line 1, in <module>
          from .more import *  # noqa
        File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/setuptools/_vendor/more_itertools/more.py", line 5, in <module>
          from concurrent.futures import ThreadPoolExecutor
        File "/tmp/pip-install-acvqye44/futures_f4c72c4008624a6b91d403bbb11b8d84/concurrent/futures/__init__.py", line 8, in <module>
          from concurrent.futures._base import (FIRST_COMPLETED,
        File "/tmp/pip-install-acvqye44/futures_f4c72c4008624a6b91d403bbb11b8d84/concurrent/futures/_base.py", line 357
          raise type(self._exception), self._exception, self._traceback
                                     ^
      SyntaxError: invalid syntax
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
Error: Process completed with exit code 1.
```

This is the section from setup.py
https://github.com/VirusTotal/vt-graph-api/blob/8f20031db833cb2cf122c7b9a0102624bc1cefa5/setup.py#L28

**Comment by aramirezmartin** [comment]
Fixed by #36

**Comment by ianhelle** [comment]
Thanks Andres. Hope you are well.
